### DISCUSSION TITLE
App Distribution options

### DISCUSSION THREAD

**Discussion by kevmoens** [dotnet/maui#646] [Q&A]
We develop line of business applications which can be hosted on-premise or in the cloud.  Currently we build desktop wpf.  We use click once to allow distribution.  We also allow for url parameters upon startup of the application and we allow extensibility of our application using MEF.  We optionally allow for biometric devices, opening of network file systems,  smart card devices and printing to label printers.  Would Maui ever be a good option for our application?  We have been looking into using blazor to share our view models so we can allow for our application to be used multi-platform with limitations.  Yet our team prefer Xaml over learning a web technology.  Any suggestions are appreciated.  Any insight into any future enhancements to Maui on any limitations would be appreciated also.

**Comment by Happypig375** [comment]
You should choose WPF if you can't wait until November to start your app.

**Reply by kevmoens** [reply]
I guess my question is will MAUI support a click once style deployment or do users have to sideload the app or use the store?
